docs/workflow/pycbc_make_coinc_search_workflow.rst

```diff
@@ -608,53 +608,250 @@ If you have a subdax that failed, ``pegasus_analyzer`` will provide you with a c
 Reuse of data from a previous workflow
 ======================================
 
-One of the features of  Pegasus is to reuse the data products of prior runs.
-This can be used to expand an analysis or recover a run with mistaken settings without
-duplicating work.
-
------------------------------------------
-Generate the full workflow you want to do
------------------------------------------
-
-First generate the full workflow for the
-run you would like to do as normal. Follow the instructions of this page from :ref:`coincworkflowgenerate`,
-but stop before planning and submitting the workflow in :ref:`coincworkflowplan`.
-
------------------------------------------------------
-Select the files you want to reuse from the prior run
------------------------------------------------------
-
-Locate the directory of the run that you would like to reuse. There is a file
-ending in ``output.map`` in the directory that you specified with the
-``--output`` argument to ``pycbc_make_coinc_search_workflow``. This file contains a 
-listing of all of the data products of the prior workflow, and can be used to tell
-pegasus to skip regenerating them.
-
-Select the entries in the file that you would like to skip generating again and
-place that into a new file. The example below selects all the inspiral and 
-tmpltbank jobs and places their entries into a new listing called prior_data.map.::
-
-    # Lets get the tmpltbank entries
-    cat /path/to/old/run/${GPS_START_TIME}-${GPS_END_TIME}/output.map | grep 'TMPLTBANK' > prior_data.map
-    
-    # Add in the inspiral  files
-    cat /path/to/old/run/${GPS_START_TIME}-${GPS_END_TIME}/output.map | grep 'INSPIRAL' >> prior_data.map
+One of the features of Pegasus is reuse the data products of prior runs.
+This can be used to e.g. expand an analysis or recover a run with mistaken settings without
+duplicating work. The steps below explain how to do this.
+
+------------------------------------
+Setting up a workflow for data reuse
+------------------------------------
+
+The first step is to generate a new workflow that performs the analysis that
+you would like to do. This workflow should be generated in a new directory so that it does not overwrite data from your previous workflows.
+Data reuse happens at the ``pycbc_submit_dax`` step, so
+first run ``pycbc_make_coinc_search_workflow`` to build a new workflow,
+following the instructions in the section :ref:`coincworkflowgenerate` of this
```

Comment:
The ref here didn't work in the compiled documentation ... Why didn't that cause a failure in the docs build??